### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by yakov116**
Wanted to make this into a lint but I don't have time

### CODE REVIEW

**Comment by fregante** [comment]
Unless these solve actual issues we have, there’s no reason to update patch version

**Comment by yakov116** [comment]
Ok I will close i will re open if I can figure out how to update tsnode

File: package.json [hunk 1]
```diff
@@ -1,6 +1,5 @@
 {
 	"private": true,
-	"type": "module",
```
Comment (yakov116) [inline package.json:3]: as per
https://github.com/TypeStrong/ts-node/discussions/1337
waiting for
https://github.com/TypeStrong/ts-node/pull/1371

**Comment by yakov116** [comment]
Blocked by https://github.com/TypeStrong/ts-node/pull/1371

File: package.json (hunk 1 above)
Comment (fregante) [reply package.json:3]: This is not a change we want, the project should stay type:module

Comment (yakov116) [reply package.json:3]: Correct that is why I closed The PR I think https://github.com/TypeStrong/ts-node/pull/1371 will fix the issue.

**Comment by yakov116** [comment]
Going to wait for more docs on this. AVA works good but I cannot figure out how to get the build to work.

File: package.json [hunk 2]
```diff
@@ -61,10 +61,10 @@
 		"github-url-detection": "^5.3.1",
 		"image-promise": "^7.0.1",
 		"indent-textarea": "^2.1.0",
-		"js-abbreviation-number": "^1.1.2",
+		"js-abbreviation-number": "^1.2.0",
 		"linkify-issues": "2.0.0-nolookbehind",
 		"linkify-urls": "3.1.0-nolookbehind",
-		"mem": "^8.1.1",
+		"mem": "^9.0.1",
```
Comment (yakov116) [inline package.json:67]: ESM 🎉

File: package.json [hunk 3]
```diff
@@ -88,40 +88,40 @@
...
+		"@types/jsdom": "^16.2.13",
+		"@types/mini-css-extract-plugin": "^2.0.1",
+		"@types/node": "^16.3.3",
+		"@types/react": "^17.0.14",
+		"@types/terser-webpack-plugin": "^5.0.4",
 		"ava": "^3.15.0",
-		"copy-webpack-plugin": "^9.0.0",
-		"css-loader": "^5.2.6",
+		"copy-webpack-plugin": "^9.0.1",
+		"css-loader": "^6.1.0",
```
Comment (yakov116) [inline package.json:102]: ESM 🎉

**Comment by yakov116** [comment]
Gave up on `ts-node` will wait a drop longer

**Comment by yakov116** [comment]
Needs another test before this is ready

**Review by fregante — APPROVED** [review]

**Comment by yakov116** [comment]
@fregante if you want to merge this, you can. I just wanted to double check that all css features are working.

**Pull request merged**